plugins/node/opentelemetry-instrumentation-graphql/test/graphql.test.ts

```diff
@@ -970,6 +972,57 @@ describe('graphql', () => {
     });
   });
 
+  describe('when specifying a responseHook configuration', () => {
+    let spans: ReadableSpan[];
+    let graphqlResult: graphqlTypes.ExecutionResult;
+    const dataAttributeName = 'graphql_data';
+
+    afterEach(() => {
+      exporter.reset();
+      graphQLInstrumentation.disable();
+      spans = [];
+    });
+
+    describe('AND valid responseHook', () => {
```

Comment:
Why the 2 describes inside the outer describe?
Perhaps give them different descriptions?